restapi.go

```diff
@@ -224,7 +225,8 @@ func (self *RestAPI) pushNotification(reqId string, kv map[string]string, perdp
 	notif := NewEmptyNotification()
 
 	for k, v := range kv {
-		if len(v) <= 0 {
+		v = strings.Trim(v, " \n")
```

Comment:
On the subject of using the trim, it seems like a client error if something sends it corrupt data like that. I would rather uniqush-push error out if something sent it bad params rather than try to make it work. It prevents client programmers from doing very silly things that get masked.